SPECS-EXTENDED/buildah/buildah.spec

```diff
@@ -89,7 +123,17 @@ make DESTDIR=%{buildroot} PREFIX=%{_prefix} -C docs install
 %dir %{_datadir}/bash-completion/completions
 %{_datadir}/bash-completion/completions/%{name}
 
+%files tests
+%license LICENSE
+%{_bindir}/%{name}-imgtype
+%{_datadir}/%{name}/test
+
 %changelog
+* Fri Dec 02 2022 Sam Meluch <sammeluch@microsoft.com> - 1.18.0-9
```

Comment:
There are minor changes to the revert in order to retain the CVE patch from 11/18.